eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -494,7 +494,7 @@ else if (valueString.charAt(0) == '\"') {
                         }
                     } catch (NullPointerException e) {
                         logger.error("The tag " + tag + " does not exist on Ewon: " +
-                            eWonName, e);
+                            eWonName + ". Check that historical logging has been enabled on the Ewon", e);
```

Comment:
This might not be related to historical logging since its in the realtime section. This comment might not be correct.

Reply:
I removed this added log.